webui/react/src/pages/Clusters/ClusterLogs.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback } from 'react';
```

Comment:
question: is there an ask for recreating this specific page? would just importing the existing page do the trick? It might help us avoid these two getting diverged in the meantime with bug fixes, new features, other changes, etc

Reply:
We might want to copy this as it's done here if we have immediate plans to change this logs page for the new design